libnetwork/iptables/iptables.go

```diff
@@ -541,6 +533,13 @@ func (iptable IPTable) raw(args ...string) ([]byte, error) {
 		commandName = "ip6tables"
 	}
 
+	if supportsXlock {
+		args = append([]string{"--wait"}, args...)
```

Comment:
iptables' `--wait` flag was introduced by this commit: https://git.netfilter.org/iptables/commit/?id=93587a04d0f2511e108bbc4d87a8b9d28a5c5dd8, which was released in v1.4.20. As pointed out in #43060, the version of iptables shipped in CentOS/RHEL 7 is v1.4.21. As such, I think we can assume `--wait` is available everywhere and ditch this `bestEffortLock`.

Reply:
Yes, I noticed that `--wait` actually was available on CentOS 7, and I wondered if it was either the wrong version mentioned, or if it was back-ported by Red Hat at some point. I didn't want to jinx it, but I was considering removing this stuff, but at least in a separate PR for visibility (in case there _would_ be issues, that would make it easier to point to)